avocado_vt/options.py

```diff
@@ -474,7 +475,12 @@ def _process_options(self):
         self.cartesian_parser = cartesian_config.Parser(debug=False)
 
         if self.options.vt_config:
-            cfg = os.path.abspath(self.options.vt_config)
+            local_cfg = os.path.abspath(self.options.vt_config)
+            cfg = data_dir.get_backend_cfg_path(self.options.vt_type,
+                                                local_cfg.split('/')[-1])
+            # Skip if cfg path is happended to local config path given
+            if local_cfg != cfg:
+                copyfile(local_cfg, cfg)
```

Comment:
This is really dangerous. Imagine you pass `tests.cfg`, `tests-shared.cfg` or `guest-os.cfg`. If I understand this correctly this way it gets overridden, right?